docs/production/authentication-methods.md

```diff
@@ -212,6 +212,55 @@ corresponding LDAP attribute is `linkedinProfile` then you just need
 to add `'custom_profile_field__linkedin_profile': 'linkedinProfile'`
 to the `AUTH_LDAP_USER_ATTR_MAP`.
 
+#### Synchronizing groups
+
+Zulip 8.0 adds preliminary support for syncing Zulip groups with LDAP groups.
```

Comment:
Let's link "Zulip groups" to https://zulip.com/help/user-groups for specificity.

We can also drop "preliminary" -- this seems like a complete implementation, and we have a habit of failing to delete those "beta" type words in documentation.

Reply:
see 20ed91f62d15e614cf0a373021c053f6ef9f6cf0